static/js/mailcheck.js

```diff
@@ -0,0 +1,281 @@
+/*
+ * Mailcheck https://github.com/mailcheck/mailcheck
+ * Author
+ * Derrick Ko (@derrickko)
+ *
+ * Released under the MIT License.
```

Comment:
Let's get this via npm; see https://zulip.readthedocs.io/en/latest/subsystems/dependencies.html for details on how to do that.